bigtop-packages/src/common/alluxio/do-component-build

```diff
@@ -19,6 +19,8 @@ set -ex
 . `dirname $0`/bigtop.bom
 
 if [ $HOSTTYPE = "powerpc64le" ] ; then
+  mvn install:install-file -DgroupId=io.grpc -DartifactId=protoc-gen-grpc-java -Dversion=1.28.1 -Dclassifier=linux-ppcle_64 -Dpackaging=exe -Dfile=/usr/src/grpc-java-1.28.0/compiler/build/exe/java_plugin/protoc-gen-grpc-java
```

Comment:
The binary of grpc 1.28.0 should be installed as io.grpc:protoc-gen-grpc-java:1.28.0 rather than 1.28.1. Then adding `-Dgrpc.version=1.28.0` to mvn options could be an option?

If grpc-1.28.0 does not work with Alluxio, you can upgrade the version of grpc added by [BIGTOP-3560](https://issues.apache.org/jira/browse/BIGTOP-3560) (#813). You need to modify bigtop-packages/src/common/zeppelin/patch1-arm64-build.diff too since it refers to the 1.28.0.